src/systems/velocity.rs

```diff
@@ -0,0 +1,9 @@
+use crate::{Position, Velocity};
+use bevy::prelude::*;
+
+pub fn velocity_system(mut query: Query<(&mut Position, &Velocity)>) {
+    for (mut position, velocity) in query.iter_mut() {
+        position.x += velocity.0.x;
+        position.y += velocity.0.y;
```

Comment:
Also I just realized that this is moving the sprites at percentage based positions rather than pixel based positions.

Meaning the ball will traverse the 60% of the screen in 1 second. I don't think we want this.